app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (OutdatedVaccineError,
+                        NotVaccinatedError,
+                        NotWearingMaskError)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    vaccines = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotWearingMaskError:
+            masks_to_buy += 1
+        except (NotVaccinatedError, OutdatedVaccineError):
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/220105914-f5ba681e-d98e-489a-b614-fd82a940e953.png)
fix this, please, also no need to create a variable, u can return a string after exception was catched